code/modules/clothing/glasses/_glasses.dm

```diff
@@ -10,13 +10,19 @@
 	resistance_flags = NONE
 	custom_materials = list(/datum/material/glass = 250)
 	var/vision_flags = 0
-	var/darkness_view = 2//Base human is 2
-	var/invis_view = SEE_INVISIBLE_LIVING //admin only for now
-	var/invis_override = 0 //Override to allow glasses to set higher than normal see_invis
+	/// Base human is 2
```

Comment:
dont convert comments into autodoc comments if they arent documentation!

Reply:
the sme goes for the rest of these they are all pretty awful

Reply:
Right, I should probably put the first two back to what they were, but the third one is legitimate in my opinion.

Reply:
well the best solution would be to make them all appropriate autodoc but sure whatever

Reply:
> well the best solution would be to make them all appropriate autodoc but sure whatever

Well, I'd do that if I actually knew what they did, but I don't I just did it quickly when I was going over it because it was annoying to not get the definition easily.